tutorials/2d/2d_movement.rst

```diff
@@ -0,0 +1,207 @@
+.. _doc_2d_movement:
+
+2D Movement Overview
+====================
+
+Introduction
+------------
+
+Every beginner has been there: "How do I move my character?" Depending on the
+style of game you're making, you may have special requirements, but in general
+the movement in most 2D games is based on a small number of 
+
+We'll use :ref:`KinematicBody2D <class_KinematicBody2D>` for these examples,
+but the principles will apply to other node types (Area2D, RigidBody2D) as well.
+
+Setup
+-----
+
+Each example below uses the same scene setup. Start with a ``KinematicBody2D`` with two
+children: ``Sprite`` and ``CollisionShape2D``. You can use the Godot icon ("icon.png")
+for the Sprite's texture or use any other 2D image you have available.
+
+Open ``Project -> Project Settings`` and select the "Input Map" tab. Add the following
+input actions (see :ref:`InputEvent <doc_inputevent>` for details):
+
+.. image:: img/movement_inputs.png
+
+8-Way Movement
+--------------
+
+In this scenario, you want the user to press the four directional keys (up/left/down/right
+or W/A/S/D) and move in the selected direction. The name "8-way movement" comes from the
+fact that diagonal movement can be chosen by pressing two keys at once.
```

Comment:
A little odd to me. The player can move diagonally by pressing 2 arrow keys simultaneously/at once.

Reply:
Reworded.